generic/Emscripten-wasm.cmake

```diff
@@ -76,5 +80,5 @@ set(CMAKE_SYSTEM_PREFIX_PATH ${CMAKE_FIND_ROOT_PATH})
 cmake_minimum_required(VERSION 3.7)
 set(CMAKE_CXX_FLAGS_INIT "-s WASM=1")
 set(CMAKE_EXE_LINKER_FLAGS_INIT "-s WASM=1")
-set(CMAKE_CXX_FLAGS_RELEASE_INIT "-DNDEBUG -O3")
-set(CMAKE_EXE_LINKER_FLAGS_RELEASE_INIT "-O3 --llvm-lto 1")
+set(CMAKE_CXX_FLAGS_RELEASE_INIT "-DNDEBUG -O3 -flto")
+set(CMAKE_EXE_LINKER_FLAGS_RELEASE_INIT "-O3 --llvm-lto 1 -flto")
```

Comment:
@mosra For your reference, here's the documentation for that: https://emscripten.org/docs/compiling/WebAssembly.html#backends

`--llvm-lto 1` is for the deprecated fastcomp backend (removed in emsc 2.0.0), `-flto(=full|thin)` is the new way of achieving link-time optimization

Reply:
Ah, found https://github.com/mosra/toolchains/issues/13, so you are already aware

Reply:
Also I was having trouble with the default build running into issues with ThinLTO. @Squareys have you gotten ThinLTO working with build? I got complaints from the wasm-ld.